src/main/java/org/cryptomator/common/mount/Mounter.java

```diff
@@ -97,7 +97,9 @@ private Runnable prepareMountPoint() throws IOException {
 				}
 			} else {
 				var mpIsDriveLetter = userChosenMountPoint.toString().matches("[A-Z]:\\\\");
-				if (!mpIsDriveLetter && canMountToParent && !canMountToDir) {
+				if (mpIsDriveLetter) {
+					if (driveLetters.getOccupied().contains(userChosenMountPoint)) throw new IllegalMountPointException("Drive \"" + userChosenMountPoint + "\" is already occupied.");
```

Comment:
This string probably needs to use a resource for translation. I'd appreciate any pointers how to do that properly, as `Mounter` does not seem to have access to the resource bundles.

Reply:
Sure:
1. Add a parameter `ResourceBundle resourceBundle` to the constructor, so you can assign it to a private final field
2. Re-run the Maven build, so that Dagger will generate the glue code used to inject the resourceBundle
3. Add english (and only english) strings to `src/main/resources/i18n/strings.properties` - all other languages will be translated later and must not be touched

Reply:
That said, this is the wrong place for translations. We don't translate exception messages but only the UI.

What I would prefer instead: Either create some specific IllegalMountPointException subclasses (we could even make this a sealed class hierarchy 🤔) or add some machine-readable cause to the exception, which can then be passed to the UI in `org.cryptomator.ui.unlock.UnlockWorkflow`.

Then, we would do the translation in this place (`org.cryptomator.ui.unlock.UnlockInvalidMountPointController`)

Reply:
Done. The only nuisance now is that the generic error dialog suggest to open the app preferences, although the misconfiguration should be resolved in the vault options.

Reply:
> We don't translate exception messages but only the UI.

Isn't this a bit of an edge-case, as here, the exception message will get displayed as part of the text in the "Unable to mount vault to custom path" dialog?

> Either create some specific IllegalMountPointException subclasses

Ok, done.

Reply:
@overheadhunter this should be good to run the GitHub workflow again.